starter-code/src/bubble-sort.js

```diff
@@ -1,12 +1,12 @@
 function bubbleSort(array) {
-    var length = array.length;
-    for (var i = (length - 1); i > 0; i--) {
+    const length = array.length;
```

Comment:
const length 
=>
const arrayLength 

Give more detailed names to your variables :)